farmdata2_modules/fd2_tabs/fd2_barn_kit/transplantingReport.html

```diff
@@ -0,0 +1,639 @@
+                 */ 
+                cropList(){
+                    let cropArray = []
+                    if(this.selectedArea != 'All'){
+                        let areaFilter = this.transplantingLogs.filter((x) => x.movement.area[0].name === this.selectedArea)
+                        cropArray = areaFilter.map(h => 
+                            this.idToCropMap.get(h.data.crop_tid)
+                        )
+                    }
+                    else{
+                        cropArray = this.transplantingLogs.map(h=> 
+                            this.idToCropMap.get(h.data.crop_tid)
+                        )
+                    }
+
+                    let crops = new Set(cropArray)
+                    let newCropArray = Array.from(crops)
+                    return newCropArray.sort()
+                },
+                /**
+                 * returns a sorted array of all the areas in the current logs. If a crop does not equal 'All' then the array will only contain areas that have the selected crop
+                 */ 
+                areaList(){
+                    let areaArray = []
+                    if(this.selectedCrop != 'All'){
+                        let cropFilter = this.transplantingLogs.filter((x) => this.idToCropMap.get(x.data.crop_tid) === this.selectedCrop)
+                        areaArray = cropFilter.map(h =>
+                            h.movement.area[0].name
+                        )
+                    }
+                    else{
+                        areaArray = this.transplantingLogs.map(h =>
+                            h.movement.area[0].name
+                        )
+                    }
+                    let areas = new Set(areaArray)
+                    let newAreaArray = Array.from(areas)
+                    return newAreaArray.sort()
+                },
+                /**
+                 * returns the total row feet for Direct Seeding
+                 */ 
+                totalRowFeet(){
+                    let tableData = this.tableRows
+                    let sumRowFeet = 0
+                    tableData.map(h => {
+                        sumRowFeet = sumRowFeet + parseInt(h.data[4])
+                    })
+                    return (Math.round(sumRowFeet*100))/100
+                },
+                /**
+                 * returns the total bed feet for Direct Seeding
+                 */ 
+                totalBedFeet(){
+                    let tableData = this.tableRows
+                    let sumBedFeet = 0
+                    tableData.map(h => {
+                        sumBedFeet = sumBedFeet + parseInt(h.data[3])
+                    })
+                    return sumBedFeet
+                },
+                /**
+                 * returns the total hours worked for direct seedings. Calculated by mulpitplying hours by workers for each log
+                 */ 
+                totalHours(){
```

Comment:
Is this calculation correct?  It looks like it is not multiplying by the number of workers.